invesalius/data/viewer_volume.py

```diff
@@ -1056,9 +1056,12 @@ def OnUpdateObjectTargetGuide(self, m_img, coord):
             if target_dist <= self.distthreshold:
                 thrdist = True
                 self.aim_actor.GetProperty().SetDiffuseColor(vtk_colors.GetColor3d('Green'))
+                if not self.show_object:
+                    self.aim_actor.GetProperty().SetOpacity(const.AIM_ACTOR_HIDDEN_OPACITY)
             else:
                 thrdist = False
                 self.aim_actor.GetProperty().SetDiffuseColor(vtk_colors.GetColor3d('Yellow'))
+                self.aim_actor.GetProperty().SetOpacity(.8)
```

Comment:
This opacity and the next ones could be taken from a value in the **constants.py** module as well. This would make it easier to tweek if needed in the future, just like you did for the other **AIM_ACTOR_HIDDEN_OPACITY**